pkg/ui/BUILD.bazel

```diff
@@ -49,6 +49,7 @@ EOF
 test_suite(
     name = "lint",
     tests = [
+        "//pkg/cmd/mirror/npm:are_lockfiles_updated",
```

Comment:
My understanding is this test will fail in CI if someone forgets to run `dev ui mirror-deps`, right?

Reply:
Correct! This shouldn't be an issue for most folks, and should only come into play for devs who add or update dependencies.